backend/api/Areas/Tools/Controllers/ImportController.cs

```diff
@@ -65,7 +66,7 @@ public ImportController(ILogger<ImportController> logger, IPimsService pimsServi
         /// <returns>The properties added.</returns>
         [HttpPost("properties")]
         [Produces("application/json")]
-        [ProducesResponseType(typeof(IEnumerable<Model.ParcelModel>), 200)]
+        [ProducesResponseType(typeof(IEnumerable<Model.ImportPropertyModel>), 200)]
```

Comment:
Used the ImportPropertyModel instead of the ParcelModel, as we need to be able to detect whether buildings were updated or added.